api/src/exam-environment/routes/exam-environment.ts

```diff
@@ -565,3 +572,28 @@ async function postScreenshotHandler(
 ) {
   return reply.code(418);
 }
+
+async function getExams(
+  this: FastifyInstance,
+  req: UpdateReqType<typeof schemas.examEnvironmentExams>,
+  reply: FastifyReply
+) {
+  const user = req.user!;
+  const exams = await this.prisma.envExam.findMany();
+
+  const availableExams = exams.map(exam => {
+    const isExamPrerequisitesMet = checkPrerequisites(user, true);
+
+    return {
+      examId: exam.id,
+      config: exam.config,
```

Comment:
We do not want the whole config. Just the `name`, `note`, and `totalTimeInMS`.

Reply:
Use `select` in the `findMany` to ensure we are explicit about what is returned. Also, so the network request is smaller.